app/presenters/link_check_csv_presenter.rb

```diff
@@ -0,0 +1,21 @@
+require 'csv'
+
+class LinkCheckCSVPresenter
+  def self.homepage_links_status_csv
+    CSV.generate do |csv|
+      csv << %w(status count)
+      LocalAuthority.group(:status).count.each do |key, value|
+        csv << [key || "nil", value]
+      end
+    end
+  end
+
+  def self.links_status_csv
+    CSV.generate do |csv|
+      csv << %w(status count)
+      Link.group(:status).count.each do |key, value|
```

Comment:
I understand, it just makes me feel slightly icky having it hanging around uncared for.  I'll worry more about it in the Autumn. :-)